src/pudl/metadata/codes.py

```diff
@@ -2318,6 +2318,160 @@
         "code_fixes": {},
         "ignored_codes": [],
     },
+    "core_eia__codes_cooling_tower_types": {
+        "df": pd.DataFrame(
+            columns=["code", "label", "description"],
+            data=[
+                (
+                    "MD",
+                    "mechanical_dry",
+                    "Mechanical draft, dry process",
+                ),
+                (
+                    "MW",
+                    "mechanical_wet",
+                    "Mechanical draft, wet process",
+                ),
+                (
+                    "ND",
+                    "natural_dry",
+                    "Natural draft, wet process",
+                ),
+                (
+                    "WD",
+                    "combo_wet_dry",
+                    "Combination wet and dry process",
+                ),
+                (
+                    "OT",
+                    "other",
+                    "Other",
+                ),
+            ],
+        )
+    },
+    "core_eia__codes_cooling_water_sources": {
+        "df": pd.DataFrame(
+            columns=["code", "label", "description"],
+            data=[
+                (
+                    "SW",
+                    "surface_water",
+                    "Surface Water (ex: river, canal, bay)",
+                ),
+                (
+                    "GW",
+                    "ground_water",
+                    "Ground Water (ex: aquifer, well)",
+                ),
+                (
+                    "PD",
+                    "plant_discharge",
+                    "Plant Discharge Water (ex: wastewater treatment plant discharge)",
+                ),
+                (
+                    "OT",
+                    "other",
+                    "Other (specify in SCHEDULE 7)",
+                ),
+            ],
+        )
+    },
+    "core_eia__codes_cooling_water_types": {
+        "df": pd.DataFrame(
+            columns=["code", "label", "description"],
+            data=[
+                (
+                    "BR",
+                    "brackish_water",
+                    "Brackish water",
+                ),
+                (
+                    "FR",
+                    "fresh_water",
+                    "Fresh water",
+                ),
+                (
+                    "BE",
+                    "reclaimed_water",
+                    "Reclaimed water (ex: treated wastewater effluent)",
+                ),
+                (
+                    "SA",
+                    "saline_water",
+                    "Saline Water",
```

Comment:
Very much a nit: Can we standardize to "water" instead of using a mix of "Water" and "water" across new coding tables?